src/fsharp/vs/service.fs

```diff
@@ -2816,10 +2847,10 @@ type FsiInteractiveChecker(reactorOps: IReactorOperations, tcConfig, tcGlobals,
         let parseResults = FSharpParseFileResults(parseErrors, inputOpt, parseHadErrors = anyErrors, dependencyFiles = dependencyFiles)
 
         let backgroundDiagnostics = []
-        let ct = CancellationToken.None
+        
         let tcErrors, tcFileResult = 
             Parser.TypeCheckOneFile(parseResults,source,mainInputFileName,"project",tcConfig,tcGlobals,tcImports,  tcState,
-                                    loadClosure,backgroundDiagnostics,reactorOps,(fun () -> true),ct,None)
+                                    loadClosure,backgroundDiagnostics,reactorOps,(fun () -> true),None) |> Async.RunSynchronously
```

Comment:
Make ParseAndCheckInteraction async please. There aren't any callers in Visual F# Tools